File: HFSGuardaDiretorio_SharpDevelop_Ruby/gui/FrmSplash.rb

require "mscorlib"
require "System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089"
require "System.Drawing, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"

module HFSGuardaDiretorio_gui

	class FrmSplash < System::Windows::Forms::Form
		def initialize()
			@components = nil
			self.InitializeComponent()
		end
		
		def InitializeComponent()
			@label1 = System::Windows::Forms::Label.new()
			@label2 = System::Windows::Forms::Label.new()
			@pb = System::Windows::Forms::ProgressBar.new()
			self.SuspendLayout()
			# 
			# label1
			# 
			@label1.Font = System::Drawing::Font.new("Microsoft Sans Serif", 20f, System::Drawing::FontStyle.Bold, System::Drawing::GraphicsUnit.Point, 0)
			@label1.Location = System::Drawing::Point.new(21, 9)
			@label1.Name = "label1"
			@label1.Size = System::Drawing::Size.new(332, 34)
			@label1.TabIndex = 0
			@label1.Text = "HFSGuardaDiretorio 2.0"
			# 
			# label2
			# 
			@label2.Font = System::Drawing::Font.new("Microsoft Sans Serif", 15f, System::Drawing::FontStyle.Bold, System::Drawing::GraphicsUnit.Point, 0)
			@label2.Location = System::Drawing::Point.new(47, 53)
			@label2.Name = "label2"
			@label2.Size = System::Drawing::Size.new(267, 29)
			@label2.TabIndex = 1
			@label2.Text = "Catalogador de Diretórios"
			# 
			# pb
			# 
			@pb.Dock = System::Windows::Forms::DockStyle.Bottom
			@pb.Location = System::Drawing::Point.new(0, 96)
			@pb.Name = "pb"
			@pb.Size = System::Drawing::Size.new(375, 17)
			@pb.Step = 1
			@pb.TabIndex = 2
			# 
			# FrmSplash
			# 
			self.AutoScaleDimensions = System::Drawing::SizeF.new(6f, 13f)
			self.AutoScaleMode = System::Windows::Forms::AutoScaleMode.Font
			self.ClientSize = System::Drawing::Size.new(375, 113)
			self.ControlBox = false
			self.Controls.Add(@pb)
			self.Controls.Add(@label2)
			self.Controls.Add(@label1)
			self.FormBorderStyle = System::Windows::Forms::FormBorderStyle.None
			self.MaximizeBox = false
			self.MinimizeBox = false
			self.Name = "FrmSplash"
			self.ShowIcon = false
			self.SizeGripStyle = System::Windows::Forms::SizeGripStyle.Hide
			self.StartPosition = System::Windows::Forms::FormStartPosition.CenterScreen
			self.Text = "FrmSplash"
			self.ResumeLayout(false)
		end
		
		def Dispose(disposing)
			if disposing then
				if @components != nil then
					@components.Dispose()
				end
			end
			self.Dispose(disposing)
		end
		
	end
end

File: HFSGuardaDiretorio_SharpDevelop_Python/gui/FrmSplash.py

import System.Drawing
import System.Windows.Forms

from System.Drawing import *
from System.Windows.Forms import *

class FrmSplash(Form):
	def __init__(self):
		self.InitializeComponent()
		
	def InitializeComponent(self):
		self._label1 = System.Windows.Forms.Label()
		self._label2 = System.Windows.Forms.Label()
		self._pb = System.Windows.Forms.ProgressBar()
		self.SuspendLayout()
		# 
		# label1
		# 
		self._label1.Font = System.Drawing.Font("Microsoft Sans Serif", 20, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, 0)
		self._label1.Location = System.Drawing.Point(21, 9)
		self._label1.Name = "label1"
		self._label1.Size = System.Drawing.Size(332, 34)
		self._label1.TabIndex = 0
		self._label1.Text = "HFSGuardaDiretorio 2.0"
		# 
		# label2
		# 
		self._label2.Font = System.Drawing.Font("Microsoft Sans Serif", 15, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, 0)
		self._label2.Location = System.Drawing.Point(47, 53)
		self._label2.Name = "label2"
		self._label2.Size = System.Drawing.Size(267, 29)
		self._label2.TabIndex = 1
		self._label2.Text = "Catalogador de Diretórios"
		# 
		# pb
		# 
		self._pb.Dock = System.Windows.Forms.DockStyle.Bottom
		self._pb.Location = System.Drawing.Point(0, 96)
		self._pb.Name = "pb"
		self._pb.Size = System.Drawing.Size(375, 17)
		self._pb.Step = 1
		self._pb.TabIndex = 2
		# 
		# FrmSplash
		# 
		self.AutoScaleDimensions = System.Drawing.SizeF(6, 13)
		self.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font
		self.ClientSize = System.Drawing.Size(375, 113)
		self.ControlBox = False
		self.Controls.Add(self._pb)
		self.Controls.Add(self._label2)
		self.Controls.Add(self._label1)
		self.FormBorderStyle = System.Windows.Forms.FormBorderStyle.None
		self.MaximizeBox = False
		self.MinimizeBox = False
		self.Name = "FrmSplash"
		self.ShowIcon = False
		self.SizeGripStyle = System.Windows.Forms.SizeGripStyle.Hide
		self.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen
		self.Text = "FrmSplash"
		self.ResumeLayout(False)